lib/syskit/cli/log_runtime_archive.rb

```diff
@@ -12,17 +13,23 @@ module CLI
         # It depends on the syskit instance using log rotation
         class LogRuntimeArchive
             DEFAULT_MAX_ARCHIVE_SIZE = 10_000_000_000 # 10G
+            FREE_SPACE_LOW_LIMIT = 1_000_000_000 # 1gb
+            FREE_SPACE_FREED_LIMIT = 10_000_000_000 # 10gb
 
             def initialize(
                 root_dir, target_dir,
                 logger: LogRuntimeArchive.null_logger,
-                max_archive_size: DEFAULT_MAX_ARCHIVE_SIZE
+                max_archive_size: DEFAULT_MAX_ARCHIVE_SIZE,
+                free_space_low_limit: FREE_SPACE_LOW_LIMIT,
+                free_space_delete_until: FREE_SPACE_FREED_LIMIT
             )
                 @last_archive_index = {}
                 @logger = logger
                 @root_dir = root_dir
                 @target_dir = target_dir
                 @max_archive_size = max_archive_size
+                @free_space_low_limit = free_space_low_limit
+                @free_space_delete_until = free_space_delete_until
```

Comment:
Validate that `free_space_low_limit > free_space_delete_until`

Reply:
I added this condition: [`return if free_space_low_limit >= free_space_delete_until`](https://github.com/rock-core/tools-syskit/pull/386/files#diff-27a24ddb639c1c67b459feffe67d0e7556ab8a2dba9de22165c32aef63d9fb4bR62)

Reply:
That's not validation, a validation statement would throw in case if was false.

About the low_limit/delete_until relation, what do you guys have in mind? I thought if low_limit < delete_until, it would be a scenario like (which is examplified by the defaults of both of these variables):
```
low_limit = 10G
delete_until = 15G

if available < low_limit
     delete files until there is 15G available
end
```

But if low_limit > delete until, then once it reaches the low limit, it would stay in a cycle deleting files until some outside actor frees enough disk to reach > low limit again.

Reply:
done